Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1026,11 +1025,11 @@ extension BrowserViewController: BrowserDelegate {
     }
 
     private func adjustFooterSize(top: UIView? = nil) {
-        snackBars.snp_remakeConstraints({ make in
+        snackBars.snp_remakeConstraints(closure: { make in
```

Comment:
Nit: We can remove the closure parameter name as well as the ( , ) and leave this as a trailing closure